cloud/new_ec2_client.go

```diff
@@ -391,5 +453,8 @@ func (c *awsClientMock) GetInstanceInfo(id string) (*ec2.Instance, error) {
 	instance.Placement.AvailabilityZone = makeStringPtr("us-east-1a")
 	instance.InstanceType = makeStringPtr("m3.4xlarge")
 	instance.LaunchTime = makeTimePtr(time.Now())
+	instance.PublicDnsName = makeStringPtr("public_dns_name")
```

Comment:
is this just a placeholder? how come it's needed?

Reply:
It's returned from the mock when testing this function https://github.com/bsamek/evergreen/blob/ac343b37e85b9634d66448a103184db34450e16e/cloud/new_ec2.go#L534.